Harvest.Web/ClientApp/src/Closeout/CloseoutContainer.tsx

```diff
@@ -0,0 +1,187 @@
+import React, { useEffect, useState } from "react";
+import { useHistory, useParams } from "react-router-dom";
+import { Label, Input, Button, UncontrolledTooltip } from "reactstrap";
+import * as yup from "yup";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPlus, faCheck } from "@fortawesome/free-solid-svg-icons";
+
+import { Project, Result } from "../types";
+import { UnbilledExpensesContainer } from "../Expenses/UnbilledExpensesContainer";
+import { ProjectHeader } from "../Shared/ProjectHeader";
+import { usePromiseNotification } from "../Util/Notifications";
+import { ProjectProgress } from "../Projects/ProjectProgress";
+import { useInputValidator } from "../FormValidation";
+import { roundToTwo } from "../Util/Calculations"
+import { useConfirmationDialog } from "../Shared/ConfirmationDialog";
+import { millisecondsPerDay } from "../Util/Calculations";
+import { ShowFor } from "../Shared/ShowFor";
+
+
+interface RouteParams {
+  projectId?: string;
+}
+
+interface FinalAcreageExpense {
+  amount: number;
+}
+
+const finalAcreageExpenseSchema = yup.object().shape({
```

Comment:
Shouldn't this be in the schema file?

Reply:
I figured it was small schema that would not be referenced anywhere else. But I can move it.

Reply:
yea might as well for consistency